app/celery/nightly_tasks.py

```diff
@@ -394,3 +395,13 @@ def delete_unneeded_notification_history_for_specific_hour(start_datetime: str,
     )
 
     delete_notification_history_between_two_datetimes(start_datetime, end_datetime)
+
+
+@notify_celery.task(name="update-report-status-to-deleted")
```

Comment:
Looking at the other nightly tasks, they all tend to have Cronitor enabled, so perhaps good to do here too?